lib/linguist/languages.yml

```diff
@@ -216,6 +216,20 @@ Alpine Abuild:
   codemirror_mode: shell
   codemirror_mime_type: text/x-sh
   language_id: 14
+Altium Designer:
+  type: data
+  aliases:
+  - altium
+  extensions:
+  - ".OutJob"
+  - ".PcbDoc"
+  - ".PrjPCB"
+  - ".SchDoc"
+  interpreters:
+  - Altium Designer
```

Comment:
Can these files really have [shebangs](https://en.wikipedia.org/wiki/Shebang_(Unix))?

Reply:
Good catch - they can't. I've removed the "interpreters" parameter for Altium and updated the PR. It wasn't clear to me which parameters were required and which were optional, so I took a stab at filling in a reasonable value there without realizing I could just omit it.